apps/meteor/client/views/room/MessageList/hooks/useMessages.ts

```diff
@@ -26,16 +29,24 @@ export const useMessages = ({ rid }: { rid: IRoom['_id'] }): MessageWithMdEnforc
 		const parseOptions = {
 			colors: showColors,
 			emoticons: true,
-			...(Boolean(katex) && {
+			...(katexEnabled && {
 				katex: {
-					dollarSyntax: katex?.dollarSyntaxEnabled,
-					parenthesisSyntax: katex?.parenthesisSyntaxEnabled,
+					dollarSyntax: katexDollarSyntaxEnabled,
+					parenthesisSyntax: katexParenthesisSyntaxEnabled,
 				},
 			}),
 		};
 		return (message: IMessage): MessageWithMdEnforced =>
-			parseMessageTextToAstMarkdown(removePossibleNullMessageValues(message), parseOptions, autoTranslateLanguage, useShowTranslated);
-	}, [autoTranslateLanguage, katex, showColors, useShowTranslated]);
+			parseMessageTextToAstMarkdown(removePossibleNullMessageValues(message), parseOptions, autoTranslateOptions);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'll add this again, but is there a reason why we need to disable this eslint rule?

Reply:
Not apparently, removing it here